### ISSUE TITLE
Could not load file or assembly 'SimMetrics.Net, Version=1.0.5.0

### ISSUE THREAD

**Issue by agalloch21** [microdee/mp.essentials#6]
Hi microdee,

I couldn't get mp.essentials.dll to work and all nodes are red. When I dragged mp.essentials.dll into vvvv, the log showed as below
`Exception during extracting node infos from file D:\vvvv\vvvvexe\vvvv_50beta38.1_x64\packs\mp.essentials\nodes\plugins\mp.essentials.dll: Could not load file or assembly 'SimMetrics.Net, Version=1.0.5.0, Culture=neutral, PublicKeyToken=c58dc06d59f3391b' or one of its dependencies. The system cannot find the file specified.`

version: vvvv_50beta38.1_x64

Could you guide me to solve this? Thank you.

**Comment by microdee** [comment]
Did you install mp.essentials or the thing depending on it via VPM from https://vvvvpm.github.io ?

**Comment by agalloch21** [comment]
Yes. the version I installed is
`github.latest.vpack  2018.09.17 10:03`

I can compile mp.essential from source code but still can't use it in vvvv.

**Comment by microdee** [comment]
ok, in your packs folder is there an md.core folder with bunch of dll's in it, including SimMetrics one? VPM should have taken care of that folder and its contents.

**Comment by agalloch21** [comment]
nope. but i installed md.core manually and now it's working. sorry to disturb you with this small issue. 
and 
thank you!

**Comment by microdee** [comment]
hmm that's not a small issue! it's a serious problem either with VPM or with the C# dependency installer scripts. Can you please try again and create a screenshot of VPM, with as much visible as possible from the end?

**Comment by agalloch21** [comment]
sorry I didn't check the message afterwards. ok, i will write down what i have done

that day, 
1. first I tried to install mp.essentials through vpm, but vpm returned some errors when installing md.stdl
```
Compilation error:
(8,9): error CS0246: The type or namespace name 'Newtonsoft' could not be found (are you missing a using directive or an assembly reference?)
(68,19): error CS0103: The name 'JArray' does not exist in the current context
```

2. the solution to solve this problem is well demonstrate by you in this [thread](https://discourse.vvvv.org/t/vpm-dx11-pack-compilation-error/15993), but the website was down that day so I couldn't reach this thread. so I tried to install mp.essentials and its dependency myself.

3. I made a mistake that I put all dependency in separated folder instead of one "md.core" folder, then I got errors when using mp.essentials in vvvv
```
Exception during extracting node infos from file D:\vvvv\vvvvexe\vvvv_50beta38.1_x64\packs\mp.essentials\nodes\plugins\mp.essentials.dll: Could not load file or assembly 'SimMetrics.Net, Version=1.0.5.0, Culture=neutral, PublicKeyToken=c58dc06d59f3391b' or one of its dependencies. The system cannot find the file specified.
```

4. the second day, I reinstalled vpm with 7zip and then reinstalled  "md.core" with vpm. it worked!
